octobot_trading/util/config_util.py

```diff
@@ -63,6 +65,29 @@ def get_symbols(config, enabled_only) -> list:
     return []
 
 
+def get_symbol_trading_type(symbol) -> str:
+    parsed_symbol = symbol_util.parse_symbol(symbol)
+    if parsed_symbol.is_spot():
+        return trading_enums.ExchangeTypes.SPOT.value
+    elif parsed_symbol.is_perpetual_future():
+        if parsed_symbol.is_linear():
+            return trading_enums.FutureContractType.LINEAR_PERPETUAL.value
+        if parsed_symbol.is_inverse():
+            return trading_enums.FutureContractType.INVERSE_PERPETUAL.value
+    else:
+        if parsed_symbol.is_linear():
+            return trading_enums.FutureContractType.LINEAR_EXPIRABLE.value
+        if parsed_symbol.is_inverse():
+            return trading_enums.FutureContractType.INVERSE_EXPIRABLE.value
+
+
+def get_symbol_types_counts(config, enabled_only) -> dict:
```

Comment:
Where is it used?

Reply:
I found it https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/997/files#diff-0facf607e31f87626ab5675ff24e7d4b1fe60449c55c29543b7c7e6e5e60daa8R81 :)